README.md

```diff
@@ -167,11 +167,14 @@ the [`JWT` authentication type](https://trino.io/docs/current/security/jwt.html)
 
 ### OAuth2 Authentication
 
-- `OAuth2Authentication` class can be used to connect to a Trino cluster configured with
+The `OAuth2Authentication` class can be used to connect to a Trino cluster configured with
 the [OAuth2 authentication type](https://trino.io/docs/current/security/oauth2.html).
-- A callback to handle the redirect url can be provided via param `redirect_auth_url_handler`, by default it just outputs the redirect url to stdout.
 
-* DBAPI
+A callback to handle the redirect url can be provided via param `redirect_auth_url_handler`, by default it will try to launch a web browser to go through the authentication flow and output the redirect url to stdout.
+
+The OAuth2 token will be cached either per `trino.auth.OAuth2Authentication` instance or, when keyring is installed, it will be cached within a secure backend (macos keychain, windows credential locker, etc) under a key including host of the Trino connection. Keyring can be installed using `pip install trino['secure-local-storage']`.
```

Comment:
nit: MacOS and Windows (instead of all lowercase)

Reply:
`pip install 'trino[secure-local-storage]'` Existing syntax with quotes inside braces fails for me.

Reply:
Yes it fails on zsh terminals, see https://stackoverflow.com/a/30539963